content/tutorials/manage-the-rippled-server/configure-peering/use-a-peer-reservation.md

```diff
@@ -0,0 +1,167 @@
+# Use a Peer Reservation
+
+A [peer reservation][] is a setting that makes a `rippled` server always accept connections from a peer matching the reservation. The following steps describe how to use peer reservations to maintain a consistent connection from one server to a specific peer, with the permission of the peer server's administrator.
```

Comment:
Not quite either one.

Which side actually benefits more from a peer reservation depends partially on which side is busier. But also, if it's your own server, you can just force it to try to connect to the peer (as described in this tutorial) and it'll do that... but it only works if the peer has an available slot ready for you, or a reservation.

I wrote this tutorial with the assumption that, for the most part, the "server" following this tutorial would be an ordinary server making an outgoing connection, while the "peer" might be a busy or (semi-)private hub receiving the incoming connection. So it's the peer—the busy server—that would need to add the reservation.

You _can_ also add the reservation on your own server to make sure it saves an outgoing slot, and arguably it's a good idea to add that step to this tutorial, but unless I'm misunderstanding something, you _do_ need the peer's admin to add the reservation to the peer server.

Reply:
> I wrote this tutorial with the assumption that, for the most part, the "server" following this tutorial would be an ordinary server making an outgoing connection, while the "peer" might be a busy or (semi-)private hub receiving the incoming connection. So it's the peer—the busy server—that would need to add the reservation.

If this is true, then it conflicts with the wording of the first sentence. It should be reworded to say:

> A peer reservation is a setting that makes a rippled ~server~ peer [the hub] always accept connections from a ~peer~ server [the reader's server] matching the reservation.

However, this wording clashes with the etymology of Peer Reservation, which is named from the perspective of the hub. In that context, the reader's server is called "peer", not the hub. This meaning matches the original wording of the first sentence.

I think the right fix here is to introduce some names other than "peer" and "server". Because they are symmetric, their meaning depends on the reader's perspective, and there seem to be some perspective shifts in the text.

I would change "with the permission of" to "with the cooperation of". The first makes me think I'm going to run an admin command on someone else's server.

Reply:
One way to clarify some of the potential confusion here might be to add a brief list of terms to the beginning of this article (we should also, at some point, create an overall glossary).

Reply:
Another option would be to change the terms used through the document. Maybe "local peer" and "remote hub".

Reply:
Those terms sound reasonable to me, @ximinez but I don't want to weigh in on the color of bike shed. I'm just saying should put an easy-to-read sign on it :) The problem now, I think, is that we're asking the reader to infer some of these definitions and concepts; we should explicitly  define them. 

Also, diagrams are usually helpful for these kinds of things. That's outside the scope of this PR--but maybe worth adding to the backlog, @mDuo13.

Reply:
I agree with John's idea that we need to introduce terms other than "peer" and "server" since those are symmetric and also overlapping. After some though, though, I decided that "local" and "remote" have the same perspective/symmetry problem that "peer" has.

I also think Ryan is on the right track with providing a glossary. So here's what I did:

- I chose the terms **stock server** and **hub server**, where the stock server makes the outgoing connection and the hub server receives the incoming connection.
- I added an explicit description at the beginning defining how I'm using these terms and what configurations these instructions describe.